Remotion/Web/Test/Startup.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Web;
+using System.Web.Integration;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Remotion.Web.Test
+{
+  public class Startup
```

Comment:
I'm missing the global.asax-code in the .net 5 application. What did I miss?

Reply:
What do you mean? The global.asax and its code-behind still exists

Reply:
ah, okay, that didn't register since it wasn't part of the diff.